rhc-ose-ansible/playbooks/openstack/terminate.yml

```diff
@@ -8,7 +8,7 @@
 - hosts: localhost
   gather_facts: false
   vars:
-    ansible_ssh_user: root
+    ansible_ssh_user: cloud-user
```

Comment:
This terminate playbook doesn't use an inventory file but it could I suppose. I've been specifying "ansible_ssh_user=cloud-user" manually via -e for a while since we changed over to the regular cloud image. I only used the root user for the OS1 images that used root, so far I've used cloud-user in our new lab. We can discuss more to decide what the default should be.